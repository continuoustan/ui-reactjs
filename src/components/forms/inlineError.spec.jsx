import React from 'react';
import renderer from 'react-test-renderer';

import InlineError from './inlineError';

test('Inline Error Display', () => {

	const props = {meta:
					{touched: true, error: true},
					required: true,
					className: 'className'
				  };
    const tree = renderer.create(
        <InlineError {...props}>
				The Content
        </InlineError>
    ).toJSON();

    expect(tree).toMatchSnapshot();
});
