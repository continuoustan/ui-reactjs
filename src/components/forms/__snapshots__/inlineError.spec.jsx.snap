// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inline Error Display 1`] = `
<div
  className="invalid className"
>
  The Content
  <div
    className="error"
  />
</div>
`;
